confluent_kafka/src/confluent_kafka.c

```diff
@@ -1146,7 +1143,7 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 			Py_DECREF(ks);
 			continue;
 
-		} else if (!strcmp(k, "error_cb")) {
+		} else if (!strcmp(k, "error_cb") && PyCallable_Check(vo)) {
```

Comment:
It will need to check PyCallable inside error_cb if-clause.
E.g.:
```
if (k=="error_cb") {
  if (!callable) {
    error_out("\"error_cb\" must be a callable");
    return..
  }
  h->error_cb = ..;
}.

```

Same thing for stats_cb